test/benchmarks/benchmark_crc32.cc

```diff
@@ -56,8 +56,13 @@ class crc32: public benchmark::Fixture {
     } \
     BENCHMARK_REGISTER_F(crc32, name)->Arg(1)->Arg(8)->Arg(12)->Arg(16)->Arg(32)->Arg(64)->Arg(512)->Arg(4<<10)->Arg(32<<10)->Arg(256<<10)->Arg(4096<<10);
 
-BENCHMARK_CRC32(braid, PREFIX(crc32_braid), 1);
-BENCHMARK_CRC32(generic, PREFIX(crc32_c), 1);
+#ifndef WITHOUT_CHORBA
+  BENCHMARK_CRC32(generic_chorba, crc32_c, 1);
```

Comment:
4 spaces?

Reply:
Removed indentation to match the rest of the file. I was thinking 1-2 spaces helped readability, 4 spaces to me instead indicates the line is indentated due to being inside a code block, so better to remove it.
I'd be in favor of adding 1-2 spaces to these for readability, but it should probably be done wholesale.